Sources/NestKit/Assets/AssetRegistryClient.swift

```diff
@@ -38,10 +46,12 @@ public struct Asset: Sendable, Equatable {
 
 public enum AssetRegistryClientError: LocalizedError, Hashable, Sendable {
     case notFound
+    case noMatchApplyingExcludedTarget
 
     public var errorDescription: String? {
         switch self {
         case .notFound: "Not found for the repository."
+        case .noMatchApplyingExcludedTarget: "Not match applying excluded version."
```

Comment:
I've changed `errorDescription` to use `version` instead of `target` for better clarity.